AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -243,13 +248,18 @@ - (void)createWithPartition:(NSString *)partition
                                                                  }
                                                                  MSLogDebug([MSDataStore logTag], @"Document json:%@", json);
 
+                                                                 // Create an instance of Document.
+                                                                 // TODO fix the warning.
```

Comment:
Is there any reason not to fix it now?

Reply:
It's "Messaging unqualified id" that because `[aClass alloc]` needs to be cast to `id<MSSerializableDocument>`.